### PR TITLE
[9.x] PHP 8, switch to match

### PR DESCRIPTION
**Pull request by mrrazvi**
much shorter and nicer way

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
This is actually a breaking change. The old code used to work with stringable objects. The new code does not.

**Comment by rizimore** [comment]
So, I close the pull request?

File: src/Illuminate/Notifications/resources/views/email.blade.php [hunk 1]
```diff
@@ -19,14 +19,10 @@
...
 <?php
-    switch ($level) {
-        case 'success':
-        case 'error':
-            $color = $level;
-            break;
-        default:
-            $color = 'primary';
-    }
+$color = match ($level) {
```
Comment (netpok) [inline src/Illuminate/Notifications/resources/views/email.blade.php:22]: To fix the BC:
```suggestion
$color = match ((string)$level) {
```

File: src/Illuminate/Notifications/resources/views/email.blade.php [hunk 2]
```diff
@@ -19,14 +19,10 @@
...
 <?php
-    switch ($level) {
-        case 'success':
-        case 'error':
-            $color = $level;
-            break;
-        default:
-            $color = 'primary';
-    }
+$color = match ((string)$level) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/resources/views/email.blade.php:22]: missing space after `(string)`

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Pull request closed without merging**